CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [12.2.0](https://github.com/theforeman/puppet-candlepin/tree/12.2.0) (2022-02-03)
+
+[Full Changelog](https://github.com/theforeman/puppet-candlepin/compare/12.1.0...12.2.0)
+
+**Implemented enhancements:**
+
+- Fixes [\#30366](https://projects.theforeman.org/issues/30366) - Candlepin config file owned by root [\#213](https://github.com/theforeman/puppet-candlepin/pull/213) ([ekohl](https://github.com/ekohl))
+- Disable FIPS support within Tomcat Java environment [\#211](https://github.com/theforeman/puppet-candlepin/pull/211) ([ehelms](https://github.com/ehelms))
```

Comment:
I'm not sure if this is a good description. It implies FIPS is always disabled, but really it provides an option to. Thoughts on changing this?

Reply:
"Disable FIPS by default within…"

IMHO "by default" implies it can be reverted?

Reply:
So like this?